packages/gatsby-theme-docs/src/components/card.js

```diff
@@ -23,6 +23,11 @@ const Container = styled.li`
   padding: ${designSystem.dimensions.spacings.m};
   border-radius: ${designSystem.tokens.borderRadiusForCard};
 `;
+const ImageContainer = styled.div`
+  svg {
+    width: 100%;
+  }
```

Comment:
Currently, SVG image is centered instead of filling the width of the card. I found a solution [here](https://stackoverflow.com/a/9573878/2467003) but that means a `viewBox` needs to be included in the SVG.